docs/index.md

```
@@ -0,0 +1,855 @@
+
+**Step 4: Address CI failures**
+```bash
+# If build_check.yml or test.yml fails, fix locally
+git add .
+git commit -m "fix: address CI failures"
+git push
+
+# CI will automatically re-run on new commit
+```
+
+**Step 5: Merge after approval**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)